lib/confirmation_text_sender.rb

```diff
@@ -0,0 +1,24 @@
+require 'rubygems'
+require 'twilio-ruby'
+require 'time'
+
+class ConfirmationTextSender
+
+  def send_message(message)
+
+    account_sid = 'AC9418dc2fc0bb5cbb33e505c227d3a3c1'
+    auth_token = '5490266efdf7640c6e1d59a7da563967'
+    @client = Twilio::REST::Client.new account_sid, auth_token
```

Comment:
you're reassigning `@client` on every call to this method, so it shouldn't be an instance variable.  A better alternative would be to lazy instantiate this in a separate method.